tests/packagedcode/test_rpm.py

```diff
@@ -391,3 +391,7 @@ def test_rpm_tags_xsetup_0_28_b1_src_rpm(self):
     def test_rpm_tags_zziplib_0_11_15_3sf_i586_rpm(self):
         test_file = self.get_test_loc('rpm/header/zziplib-0.11.15-3sf.i586.rpm')
         self.check_rpm_tags(test_file)
+
+    def test_rpm_tags_apache_commons_io_2_4_12_el7_noarch_rpm(self):
```

Comment:
Maybe `test_get_rpm_tags_can_parse_numeric_values` is a better name?

Reply:
the tests above also check for the same function right? 
Should I update those too?